lib/filesystem/FileStream.cpp

```diff
@@ -0,0 +1,148 @@
+#include "StdInc.h"
+#include "FileStream.h"
+#include "../minizip/unzip.h"
```

Comment:
I tried that, but then compilation fails. There's a strange macro in the minizip headers which doesn't get defined the right way if I only include "ioapi.h", resulting in compilation errors.